include/Makefile.am

```diff
@@ -21,16 +21,14 @@ noinst_HEADERS = \
 	nccl_ofi_memcheck.h \
 	nccl_ofi_memcheck_nop.h \
 	tracepoint.h \
-	nccl-headers/error.h \
 	nccl-headers/net.h \
-	nccl-headers/net_neuron.h \
-	nccl-headers/net_v2.h \
-	nccl-headers/net_v3.h \
-	nccl-headers/net_v4.h \
-	nccl-headers/net_v5.h \
-	nccl-headers/net_v6.h \
```

Comment:
Tried a Neuron build of this PR, and there are some files still using the older include paths/header names.

Reply:
Fixed.